scripts/stack-compose-runtime.sh

```
@@ -183,6 +188,42 @@ arr_safe_compose_write() {
   return 1
 }
 
+arr_compose_resolve_lan_binding_host() {
+  local __dest_var="$1"
+  local candidate="${2:-${LAN_IP:-}}"
+  local placeholder_host='${LAN_IP}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)